src/core/reducers/data/navigations.ts

```diff
@@ -54,7 +48,7 @@ export const updateSearch = (state: State, payload: Actions.Payload.Search) => {
     } else {
       return addRefinement({ ...state, byId }, <Actions.Payload.Navigation.AddRefinement>payload);
     }
-  } else {
+  } else if (payload.clear) {
```

Comment:
it looks like we're missing one of the test cases here as we only have partial coverage of this line